src/tyr/route_serializer_osrm.cc

```diff
@@ -546,6 +548,15 @@ json::ArrayPtr intersections(const valhalla::DirectionsLeg::Maneuver& maneuver,
 
         if (routeable && intersecting_edge->use() == TripLeg_Use_kRestAreaUse) {
           rest_stop->emplace("type", std::string("rest_area"));
+          if (intersecting_edge->has_sign()) {
```

Comment:
Here's where we check if the `intersecting_edge` has a sign, and if so, grab the name of the sign and add it to the response.